ios/iNaturalistReactNative/Info.plist

```diff
@@ -73,6 +73,8 @@
 	<string>Export iNaturalist photos to your library.</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Export and import iNaturalist photos to and from your library.</string>
+	<key>RCTNewArchEnabled</key>
+	<true/>
```

Comment:
what's the significance of this? I might have misunderstood but I thought we were already on new arch